assets/i18n/en.json

```diff
@@ -92,6 +92,9 @@
     "I18N_ANDROID_PAGE_UPDATES_SUBMIT_BUTTON_TEXT": "Notify Me",
     "I18N_ANDROID_PAGE_UPDATES_SUBTEXT": "And want to hear when the app is available<br>in your language? <span class=\"oppia-android-bold-text\">We'll notify you!</span>",
     "I18N_ANDROID_PAGE_VERSIONS_SUPPORTED_TEXT": "We support all Android versions going back to Lollipop (Android 5).",
+    "I18N_ANSWER_MISSPELLED_RESPONSE_TEXT_0": "You're headed in the right direction, but you need to recheck your spelling.",
+    "I18N_ANSWER_MISSPELLED_RESPONSE_TEXT_1": "You’re close to the right answer. Could you please correct your spelling?",
+	"I18N_ANSWER_MISSPELLED_RESPONSE_TEXT_2": "Please recheck your spelling.",
```

Comment:
Extra indentation.

Reply:
This indentation doesn't actually appear in VS Code. So, I am a bit confused here.

Reply:
There might be some issue in your editor then. You can confirm by looking at the file directly [here](https://github.com/oppia/oppia/blob/9bd68c52133b6b97a0f53df9e909820e478c8621/assets/i18n/en.json).